### ISSUE TITLE
Make Memory display the Active percentage instead of Used

### ISSUE THREAD

**Issue by lepa22** [AstraExt/astra-monitor#9]
## Description
The Memory icon in top bar currently displays the *Used* memory. I think it is better to show the *Active* memory instead, or at least there should be an option for what should be displayed. The reason is that *Used* memory is the total memory of active and inactive processes, while *Active* memory is the memory used by active processes only. The memory of inactive processes just sits in RAM to be used by these processes if they become active again, however, it can be allocated by the system to other processes depending on the system's needs. It is in other words idle memory.

For example, a user opening applications and closing them will increase the memory usage of inactive processes. Depending on the number of applications opened and closed and on the system's RAM, inactive processes can make the *Used* memory go to  100%, even though the *Active* memory would in reality be much lower. This could alarm users falsely thinking that there is a memory issue with their system, while in reality things would function normally. Also, System Monitor and most system monitoring utilities display the *Active* memory instead of *Used* memory.

In conclusion, I think that it is preferable to show by default the *Active* memory on the top bar instead of the *Used* memory which reports higher memory usage that can be misleading.

(In the above I have used the terms *Used* and *Active* in the context of your extension. Other tools, such as `free`, might use different terms for the corresponding memory usage type.)

## Environment
- **Astra Monitor Version**: v2 (as shown on GitHub)
- **GNOME Version**: 45.3
- **Operating System and Version**: Fedora 39 Workstation

**Comment by ljuzig** [comment]
Customization of memory usage indicator was already on my list, it is indeed confusing right now. I will try to make it fit for the next release. Thank you!
